src/crypto-api.ts

```diff
@@ -137,6 +144,16 @@ export interface CryptoApi {
      */
     isCrossSigningReady(): Promise<boolean>;
 
+    /**
+     * Get the ID of one of the user's cross-signing keys.
+     *
+     * @param type - The type of key to get the ID of.  One of
+     *     "master", "self_signing", or "user_signing".  Defaults to "master".
+     *
+     * @returns If cross-signing has been initialised on this device, the ID of the given key. Otherwise, null
+     */
+    getCrossSigningKeyId(type?: CrossSigningKey | string): Promise<string | null>;
```

Comment:
Can we use a const enum instead of this union?

Reply:
actually we don't need this union any more.
```suggestion
    getCrossSigningKeyId(type?: CrossSigningKey): Promise<string | null>;
```